<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UWB科技HR-RTLS上位机</string>
  </property>
  <widget class="GraphicsWidget" name="graphicsWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="action_cn"/>
    <addaction name="action_en"/>
   </widget>
   <addaction name="viewMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="minimap_dw">
   <property name="minimumSize">
    <size>
     <width>255</width>
     <height>39</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/haorutech.ico</normaloff>:/icons/haorutech.ico</iconset>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="windowTitle">
    <string>缩略底图</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="MinimapView" name="minimap"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="viewSettings_dw">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>56</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/haorutech.ico</normaloff>:/icons/haorutech.ico</iconset>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="windowTitle">
    <string>参数设置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="ViewSettingsWidget" name="viewSettings_w"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>关于UWB科技</string>
   </property>
   <property name="toolTip">
    <string>关于UWB科技</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/haorutech.ico</normaloff>:/icons/haorutech.ico</iconset>
   </property>
   <property name="text">
    <string>Channel Config</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>中/英文切换</string>
   </property>
  </action>
  <action name="action_cn">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="action_en">
   <property name="text">
    <string>英文</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ViewSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header>GraphicsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinimapView</class>
   <extends>QWidget</extends>
   <header>MinimapView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
